research/cifar10/model.py

```diff
@@ -0,0 +1,47 @@
+import torch
+import torch.nn as nn
+from torch.nn import BatchNorm2d, Conv2d, Flatten, Linear, MaxPool2d, Module, ReLU
+
+
+class ConvNet(Module):
+
+    def __init__(
+        self,
+        in_channels: int,
+        h: int = 32,
+        w: int = 32,
+        hidden: int = 2048,
+        class_num: int = 10,
+        use_bn: bool = True,
```

Comment:
Good point I will do another set of experiments by not using it.

Reply:
I'm fairly certain the benchmark consider this in their tests. I don't imagine it will significantly change the results, but it might bring them slightly closer together.